src/plugins/index_pattern_management/public/components/edit_index_pattern/indexed_fields_table/indexed_fields_table.test.tsx

```diff
@@ -28,14 +29,41 @@ jest.mock('./components/table', () => ({
 const helpers = {
   editField: (fieldName: string) => {},
   deleteField: (fieldName: string) => {},
-  getFieldInfo: () => [],
+  // getFieldInfo handles non rollups as well
+  getFieldInfo: new RollupIndexPatternListConfig().getFieldInfo,
 };
 
 const indexPattern = ({
   getNonScriptedFields: () => fields,
   getFormatterForFieldNoDefault: () => ({ params: () => ({}) }),
 } as unknown) as IndexPattern;
 
+const rollupIndexPattern = ({
+  type: IndexPatternType.ROLLUP,
+  typeMeta: {
+    'rollup-index': 'rollup',
```

Comment:
Is this `TypeMeta` interface? should then this be inside `params`?

Reply:
ah, yes!